content/projects/gsoc/2020/project-ideas/pipeline-as-yaml-experiment.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: gsocprojectidea
+title: "Pipeline: YAML Plugin"
+goal: "Jenkins Pipeline as YAML: experimental plugin"
+category: Plugins
+year: 2020
+status: draft
+sig: platform
+mentors:
+- "oleg_nenashev"
+skills:
+- Java
+- Jenkins Pipeline
+- DSL
+- Jenkins X
+links:
+  gitter: "jenkinsci/pipeline-authoring-plugin"
+---
+
+Jenkins Pipeline is a widely used way to define jobs in Jenkins.
+There are syntaxes: Declarative and Scripted Pipeline.
+Both of them are based on Groovy DSL, and there is interest in a declarative YAML syntax expressed by users.
+YAML definitions are widely used in CI/CD systems, including Jenkins X, Tekton, and many popular CI/CD services (see below).
```

Comment:
I'd suggest adding something about YAML being a widely used language, so many people would find it much easier to create pipelines using YAML than groovy. This makes Jenkins more approachable for people who don't know a lot about it.

Reply:
Could you please suggest a change for this text? Thanks!

Reply:
```suggestion
YAML definitions are widely used in CI/CD systems, including Jenkins X, Tekton, and many popular CI/CD services (see below).
Having YAML support in Jenkins would make it more approachable for people who don't know a lot about low-level Pipeline scripting.
```